Subject: [PATCH] fix mv643xx compilation                                                                                      
From: Olaf Hering <olh@suse.de>                                                                                               

linux-2.6.15/drivers/net/mv643xx_eth.c: In function `mv643xx_eth_start_xmit':
linux-2.6.15/drivers/net/mv643xx_eth.c:1159: error: dereferencing pointer to incomplete type
linux-2.6.15/drivers/net/mv643xx_eth.c:1161: error: dereferencing pointer to incomplete type
linux-2.6.15/drivers/net/mv643xx_eth.c:1161: error: `IPPROTO_UDP' undeclared (first use in this function)
linux-2.6.15/drivers/net/mv643xx_eth.c:1161: error: (Each undeclared identifier is reported only once
linux-2.6.15/drivers/net/mv643xx_eth.c:1161: error: for each function it appears in.)
linux-2.6.15/drivers/net/mv643xx_eth.c:1164: error: dereferencing pointer to incomplete type
linux-2.6.15/drivers/net/mv643xx_eth.c:1164: error: `IPPROTO_TCP' undeclared (first use in this function)
linux-2.6.15/drivers/net/mv643xx_eth.c:1222: error: dereferencing pointer to incomplete type
linux-2.6.15/drivers/net/mv643xx_eth.c:1224: error: dereferencing pointer to incomplete type
linux-2.6.15/drivers/net/mv643xx_eth.c:1227: error: dereferencing pointer to incomplete type

Signed-off-by: Olaf Hering <olh@suse.de>
 drivers/net/mv643xx_eth.c |    1 +
 1 files changed, 1 insertion(+)

Index: linux-2.6.15/drivers/net/mv643xx_eth.c
===================================================================
--- linux-2.6.15.orig/drivers/net/mv643xx_eth.c
+++ linux-2.6.15/drivers/net/mv643xx_eth.c
@@ -35,6 +35,7 @@
 #include <linux/tcp.h>
 #include <linux/udp.h>
 #include <linux/etherdevice.h>
+#include <linux/in.h>
 
 #include <linux/bitops.h>
 #include <linux/delay.h>
